AnnoyancesFilter/sections/cookies_whitelist.txt

```diff
@@ -14,6 +14,9 @@ kupbilecik.pl#@#.cookie-container
 @@||app.usercentrics.eu/browser-ui/latest/bundle.js$domain=studycheck.de
 @@||api.usercentrics.eu/settings^$domain=studycheck.de
 @@||aggregator.service.usercentrics.eu/aggregate^
+! https://github.com/AdguardTeam/AdguardFilters/issues/76611
+!+ PLATFORM(ios, ext_safari)
+@@||tommy.com^$generichide
```

Comment:
what the purpose of this rule?

Reply:
`#cookiesMessage` generic rule will remove element but modal backdrop will stay.

Reply:
why not just these?
```
tommy.com#@##cookiesMessage
tommy.com#@#.cookies-message
```